pkgs/development/python-modules/safeio/default.nix

```
@@ -0,0 +1,31 @@
+{ lib, buildPythonPackage, fetchFromGitHub, fetchpatch }:
+
+buildPythonPackage rec {
+  pname = "safeio";
+  version = "1.2";
+
+  src = fetchFromGitHub {
+    owner = "Animenosekai";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "koun/XMsXKWNEumR6lrso8u9FInRct8fPUt+2fX2Bvg=";
+  };
+
+  patches = [
+    # Adds setup.py and support files. Will be included in the next release
+    (fetchpatch {
+      url = "https://github.com/Animenosekai/safeIO/commit/36169fd16e316feb3d3f23c52881d3cadb331241.patch";
+      sha256 = "0pk61dv3h79cld0xaial57q56ccl294xcikqwnkr3cqg1646jk4r";
+    })
+  ];
+
+  doCheck = false;
```

Mentioning why `doCheck` is set to false helps to avoid duplicate work.

```suggestion
  # Module has no tests
  doCheck = false;
```